client/components/App.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```

Comment:
Cool, I only changed to .jsx so Sublime's syntax highlighting would work correctly. It was defaulting to JS syntax highlighting for me instead of JavaScript (babel).